frontend/ui/elements/screensaver_menu.lua

```diff
@@ -38,6 +38,51 @@ return {
             genMenuItem(_("Show reading progress on sleep screen"), "screensaver_type", "readingprogress", isReaderProgressEnabled),
             genMenuItem(_("Show book status on sleep screen"), "screensaver_type", "bookstatus", hasLastFile),
             genMenuItem(_("Leave screen as-is"), "screensaver_type", "disable", nil, true),
+            {
+                text = _("Ignore book cover"),
+                help_text = _("Choose when to ignore showing book covers on the sleep screen."),
+                enabled_func = function()
+                    return G_reader_settings:readSetting("screensaver_type") == "cover"
+                end,
+                checked_func = function()
+                    return G_reader_settings:isTrue("screensaver_exclude_finished_books")
+                            or G_reader_settings:isTrue("screensaver_hide_cover_in_filemanager")
+                            or G_reader_settings:isTrue("screensaver_exclude_on_hold_books")
```

Comment:
Please re-order them so hide_cover_in_filemanager doesn't end in the middle of the other more-related 2 (put fm first, so the exclude_ ones are visually aligned).